package.json

```diff
@@ -40,23 +40,34 @@
     "cli-progress": "^3.8.2",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
+    "crc": "^3.8.0",
     "encryptedfs": "^0.1.0",
     "fs-extra": "^9.0.0",
     "futoin-hkdf": "^1.3.2",
+    "git-list-pack": "0.0.10",
```

Comment:
Can we synthesize this code instead of relying on such an old third party dependency.

Reply:
So it turns out that the git backend works just fine without this so I've taken it out.